tools/accuracy_checker/accuracy_checker/launcher/onnx_launcher.py

```diff
@@ -78,10 +78,13 @@ def predict(self, inputs, metadata=None, **kwargs):
 
         return results
 
-    @staticmethod
-    def fit_to_input(data, layer_name, layout):
+    def fit_to_input(self, data, layer_name, layout):
+        layer_shape = self.inputs[layer_name]
         if len(np.shape(data)) == 4:
-            return np.transpose(data, layout).astype(np.float32)
+            data = np.transpose(data, layout).astype(np.float32)
+            if len(layer_shape) == 3:
+                return data[0]
```

Comment:
I think there should be a check (or at least an assert) that `data` has size 1 in the first dimension. Otherwise, we could silently lose data.

Reply:
added